global.d.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt The complete set of authors may be found
+ * at http://polymer.github.io/AUTHORS.txt The complete set of contributors may
+ * be found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by
+ * Google as part of the polymer project is also subject to an additional IP
+ * rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+declare namespace PolymerDecorators {
+  function customElement(tagname?: string): (clazz: any) => void;
+
+  interface PropertyOptions {
+    notify?: boolean;
```

Comment:
Add a todo to add the other options.

Reply:
Done (in `decorators.ts`).

Reply:
Note that some options are better as separate decorators, or as TypeScript features. `computed` should be a decorator on a method. `observer` is taken care of by `@observe`. `readOnly` might be better as a `@property` decorator on a getter (not 100% sure). `value` is taken care of by field initializers.